mala/interfaces/__init__.py

```diff
@@ -1,2 +1,3 @@
 """Interfaces to other codes for workflow setup (e.g. MD or MC)."""
 from .ase_calculator import MALA
+from .ase_calculator_uncertainty import MALAUncertainty
```

Comment:
Can this class be moved to the `ase_calculator` module, or what is the reason for having a separate module?

Reply:
I was under the impression that one class per file is best practice?

Reply:
Not sure if that is a best practice. It is a design descision and here it makes sense to make the separation more visible, I guess.